temporalio/worker/worker.py

```diff
@@ -279,14 +317,9 @@ async def shutdown(self) -> None:
         bridge_shutdown_task = asyncio.create_task(self._bridge_worker.shutdown())
         # Wait for the poller loops to stop
         await self._task
-        # Shutdown the workers
-        worker_tasks: List[asyncio.Task] = []
+        # Shutdown the activity worker (there is no workflow worker shutdown)
```

Comment:
Are you just assuming that the workflow worker completes when bridge worker shutdown is complete?